src/main/java/fr/openent/competences/helpers/NoteControllerHelper.java

```diff
@@ -0,0 +1,32 @@
+package fr.openent.competences.helpers;
+
+import fr.openent.competences.constants.Field;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+public class NoteControllerHelper {
+
+    protected static final Logger log = LoggerFactory.getLogger(DevoirControllerHelper.class);
```

Comment:
Pas la bonne classe du logger 

```suggestion
    protected static final Logger log = LoggerFactory.getLogger(NoteControllerHelper.class);
```